docs/developer-guide/local_testing_guide.md

```diff
@@ -0,0 +1,285 @@
+rm -rf .git
+
+sudo cp _output/local/go/bin/e2e.test /usr/local/bin/.
+sudo cp _output/local/go/bin/kubectl /usr/local/bin/kubectl-$K8S_VERSION
+sudo ln -s /usr/local/bin/kubectl-$K8S_VERSION /usr/local/bin/kubectl
+cp _output/local/go/bin/ginkgo /usr/local/bin/.
+popd
+```
+
+If you have any failures during the build, verify $PATH has been updated to
+point to correct GO version. Also may need to change settings on some of the
+generated binaries. For example:
+
+```
+chmod +x $GOPATH/src/k8s.io/kubernetes/_output/bin/deepcopy-gen
+```
+
+## Export environment variables
+
+Before setting up KIND and before running the actual tests, export essential environment variables.
+
+The environment variables and their values depend on the actual test scenario that you want to run.
+
+Look at the `e2e` action (search for `name: e2e`) in [ovn-kubernetes/.github/workflows/test.yml](https://github.com/ovn-org/ovn-kubernetes/blob/master/.github/workflows/test.yml). Prior to installing kind, set the following environment variables according to your needs:
+```
+export KIND_CLUSTER_NAME=ovn
+export KIND_INSTALL_INGRESS=[true|false]
+export KIND_ALLOW_SYSTEM_WRITES=[true|false]
+export PARALLEL=[true|false]
+export JOB_NAME=(... job name ...)
+export OVN_HYBRID_OVERLAY_ENABLE=[true|false]
+export OVN_MULTICAST_ENABLE=[true|false]
+export OVN_EMPTY_LB_EVENTS=[true|false]
+export OVN_HA=[true|false]
+export OVN_DISABLE_SNAT_MULTIPLE_GWS=[true|false]
+export OVN_GATEWAY_MODE=["local"|"shared"]
+export PLATFORM_IPV4_SUPPORT=[true|false]
+export PLATFORM_IPV6_SUPPORT=[true|false]
+# not required for the OVN Kind installation script, but export this already for later
+OVN_SECOND_BRIDGE=[true|false]
+```
+
+You can refer to a recent CI run from any pull request in [https://github.com/ovn-org/ovn-kubernetes/actions](https://github.com/ovn-org/ovn-kubernetes/actions) to get a valid set of settings.
+
+As an example for the `control-plane-noHA-local-ipv4-snatGW-1br` job, the settings are at time of this writing:
+```
+export KIND_CLUSTER_NAME=ovn
+export KIND_INSTALL_INGRESS=true
+export KIND_ALLOW_SYSTEM_WRITES=true
+export PARALLEL=true
+export JOB_NAME=control-plane-noHA-local-ipv4-snatGW-1br
+export OVN_HYBRID_OVERLAY_ENABLE=true
+export OVN_MULTICAST_ENABLE=true
+export OVN_EMPTY_LB_EVENTS=true
+export OVN_HA=false
+export OVN_DISABLE_SNAT_MULTIPLE_GWS=false
+export OVN_GATEWAY_MODE="local"
+export PLATFORM_IPV4_SUPPORT=true
+export PLATFORM_IPV6_SUPPORT=false
+# not required for the OVN Kind installation script, but export this already for later
+export OVN_SECOND_BRIDGE=false
+```
+
+## KIND
+
+Kubernetes in Docker (KIND) is used to deploy Kubernetes locally where a docker
+container is created per Kubernetes node. The CI tests run on this Kubernetes
+deployment. Therefore, KIND will need to be installed locally.
+
+Generic instructions for installing and running OVNKubernetes with KIND can be found at:
```

Comment:
OVN Kubernetes

Reply:
I had a feeling it looks wrong :D thank you!